README.md

```diff
@@ -212,6 +212,31 @@ const [user, setUser] = useState<IUser | null>(null);
 setUser(newUser);
 ```
 
+**useRef**
+
+When using `useRef`, you have two options when creating a ref container that does not have an initial value:
+
+```ts
+const ref1 = useRef<HTMLElement>(null)
+const ref2 = useRef<HTMLElement | null>(null)
+```
+
+The first option will make `ref1.current` read-only, and is intended to be passed in to built-in `ref` attributes that React will manage (because React handles setting the `current` value for you).
+
+The second option will make `ref2.current` mutable, and is intended for "instance variables" that you manage yourself.
```

Comment:
:heart_eyes: for whomever came up with this solution.

What about `const ref = useRef(someObject);`? Will this infer `Ref<typeof someObject>` or `Ref<typeof someObject | null>`?

Reply:
That was the case in `React.createRef` which doesn't have *any* parameters. Now with `useRef` you have to supply an initial value, so it's a bit different. 

If you want to use it like an instance variable it has to be `useRef<number | null>(null)` now.

I also didn't want to get too into hooks usage in general, because that would overload the goal of the cheatsheet which is to just teach TS with React, so I assume people know how to use the built-in hooks as far as runtime goes. We could add some more detail though, I'm somewhat ambivalent.